core/src/main/java/com/dtolabs/rundeck/core/execution/ExecArg.java

```diff
@@ -33,6 +33,8 @@
 
     private boolean quoted = true;
 
+    private boolean featureQuoting = false;
```

Comment:
Can we change the property name to `featureQuotingBackwardCompatible`? It's much easier to understand what's the purpose of this new add configuration item.
And please add comments above to describe the purpose of this field.

Thanks!